migrator/runner/runner.go

```diff
@@ -74,6 +77,18 @@ func Run(databases *types.Databases) error {
 
 func runMigrations(databases *types.Databases, startingSeqNum int) error {
 	for seqNum := startingSeqNum; seqNum < pkgMigrations.CurrentDBVersionSeqNum(); seqNum++ {
+		// Add an outer transaction so migrations can be wrapped in a transaction.
+		ctx = sac.WithAllAccess(context.Background())
+
+		tx, err := databases.PostgresDB.Begin(ctx)
+		if err != nil {
+			return err
+		}
+		ctx = pgPkg.ContextWithTx(ctx, tx)
+
+		// Set the context with the databases so the wrapped transaction can be used
+		databases.DBCtx = ctx
```

Comment:
It looks like we do not need to embed ctx into databases. How about passing it to migration.Run(...) as a parameter?

Reply:
That would require a change to the interface which would ripple through all the `Run` functions.  I was trying to avoid that.